lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -6901,5 +6900,15 @@ is the only supported options.</xs:documentation>
       <xs:enumeration value="aggressive"/>
     </xs:restriction>
   </xs:simpleType>
-
+  <xs:simpleType name="CollectionType">
+    <xs:annotation>
+      <xs:documentation xml:lang="en">Allowed collection types</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="list"/>
+      <xs:enumeration value="paired"/>
+      <xs:enumeration value="list:list"/>
+      <xs:enumeration value="list:paired"/>
```

Comment:
There's no limit to the allowed collection type nesting.

Reply:
Interesting. Did not know this.